### PR TITLE
UI: make header

### PR DESCRIPTION
**Pull request by harshbrahmxatriya**
This commit is to demo the header
How to test:
Go to URL: https://deploy-preview-42--wehelpvip.netlify.app/
See the header part. But note that in order to see the mobile view which is there in the figma file, you have to resize the browser window such that it represents a phone's screen, viz. resize it with least width and max height, so like a vertical rectangle like a phone screen. Then you can click the menu icon and see the slide over panel.

### CODE REVIEW

**Review by razbakov — CHANGES_REQUESTED** [review]
delete `assets/images/menu.png`

File: package.json [hunk 1]
```diff
@@ -22,6 +22,8 @@
   "dependencies": {
     "@headlessui/vue": "^1.4.2",
     "@heroicons/vue": "^1.0.5",
+    "@tailwindcss/forms": "^0.5.0",
```
Comment (razbakov) [inline package.json:25]: please remove all commits from your branch that are not relevant to your current issue.

File: app.vue [hunk 2]
```diff
@@ -1,6 +1,5 @@
 <template>
   <div>
-    <LayoutHeader />
```
[the remark this answers is not in the document]
Comment (harshbrahmxatriya) [reply app.vue:3]: OK

File: package.json (hunk 1 above)
Comment (harshbrahmxatriya) [reply package.json:25]: okay

File: app.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (razbakov) [reply app.vue:3]: Don't delete this component.

**Comment by harshbrahmxatriya** [comment]
> delete `assets/images/menu.png`

Done

**Comment by harshbrahmxatriya** [comment]
> This commit is to demo the header How to test: Go to URL: https://deploy-preview-42--wehelp-harshb.netlify.app/header

File: components/layout/header.vue [hunk 3]
```diff
@@ -1,13 +1,38 @@
 <template>
+  <!-- <div class="flex justify-between items-center p-10 shadow">
```
Comment (razbakov) [inline components/layout/header.vue:2]: Please remove unused code from your PRs.

File: components/layout/header.vue [hunk 4]
```diff
@@ -1,13 +1,38 @@
...
   <Popover class="relative bg-white">
     <div class="max-w-7xl mx-auto px-4 sm:px-6">
       <div
         class="flex justify-between items-center border-b-2 border-gray-100 py-6 md:justify-start md:space-x-10"
       >
         <div class="flex justify-start lg:w-0 lg:flex-1">
-          <NuxtLink to="/">
-            <BrandLogo class="w-24 h-24" />
-          </NuxtLink>
+          <a href="#">
```
Comment (razbakov) [inline components/layout/header.vue:28]: Here you have a logo already. Revert that change.

File: components/layout/header.vue [hunk 5]
```diff
@@ -18,21 +43,203 @@
...
+                  </div>
+                </div>
+              </PopoverPanel>
+            </transition>
+          </Popover>
+
+          <a
+            href="#"
             class="text-base font-medium text-gray-500 hover:text-gray-900"
-            v-for="item in navigation"
```
Comment (razbakov) [inline components/layout/header.vue:23]: use navigation array to define navigation. see the code that was already there ;)

File: app.vue (hunk 2 above)
[the remark this answers is not in the document]
Comment (harshbrahmxatriya) [reply app.vue]: But I have retained that component now

File: components/layout/header.vue (hunk 4 above)
Comment (harshbrahmxatriya) [reply components/layout/header.vue:28]: Should I remove the wehelp logo? Am I interpreting it right?

File: components/layout/header.vue (hunk 5 above)
Comment (harshbrahmxatriya) [reply components/layout/header.vue]: Have a look now

**Comment by razbakov** [comment]
“This commit is to demo the header” - no need to write this in PR. PR is a demo for review itself

**Pull request closed without merging**